sort/bogoSort.c

```diff
@@ -51,6 +54,7 @@ int main(void)
     unsigned long long spentTime = time(NULL) - now;
 
     printf("\nThe time spent to sort was: %llu seconds\n", spentTime);
+    printf("Number of interaction: %d\n", interaction);
```

Comment:
I think that we should use the plural to the interaction word, such as "Number of interactions: %d\n"